docs/element-definitions.md

```diff
@@ -720,7 +758,7 @@ Type Element matching only a specific property list instance (JSON `{"foo": fals
 
 ```
 
-Type Element matching only a property list with key "foo" of value type `boolean` and with the key "bar" of value type `string` (JSON `{"foo": false, "bar": "fun"}`, `{"foo": true, "bar": ""}`).
+Type Element representing only a property list with key "foo" of value type `boolean` and with the key "bar" of value type `string` (JSON `{"foo": false, "bar": "fun"}`, `{"foo": true, "bar": ""}` et cetera).
```

Comment:
I think the text is scientific enough, we can avoid literal latin expansions of pretty common abbreviations like `etc.`